airflow/www/extensions/init_views.py

```diff
@@ -323,11 +301,7 @@ def init_api_experimental(app):
     app.extensions["csrf"].exempt(endpoints.api_experimental)
 
 
-def init_api_auth_provider(app):
+def init_api_auth_provider(connexion_app: connexion.FlaskApp):
```

Comment:
I know it might not be related to this change but `auth_provider` no longer make sense, we should update it as well

```suggestion
def init_api_auth_manager(connexion_app: connexion.FlaskApp):
```